_data/projects/julia.yml

```diff
@@ -0,0 +1,9 @@
+name: Julia Language
+desc: Julia is a high-level, high-performance dynamic programming language for technical computing. https://www.julialang.org
+site: https://github.com/JuliaLang/julia 
```

Comment:
Entirely up to you whether you'd like this to point to the repository or the top-level site, but this is the "project name" link the user will click.